### PR TITLE
chore(*): Fix rating samples code view tabs and live-editing

### PR DESCRIPTION
**Pull request by dobromirts**
Related to #3358

Should be tested with: https://github.com/IgniteUI/igniteui-live-editing/pull/39

### CODE REVIEW

**Comment by kdinev** [comment]
@ChronosSF I suggest this fix to also be verified and merged before promoting to production. What do you think?

File: live-editing/configs/GridConfigGenerator.ts [hunk 1]
```diff
@@ -1445,6 +1446,28 @@ export class GridConfigGenerator implements IConfigGenerator {
...
+                ngImports: ['IgxPreventDocumentScrollModule', 'IgxGridModule', 'IgxToastModule']
+            })
+        }));
+
+        configs.push(new Config({
+            component: 'RatingInFormComponent',
+            additionalFiles: [],
+            additionalDependencies: ['igniteui-webcomponents'],
+            appModuleConfig: new AppModuleConfig({
+                imports: ['IgxButtonModule', 'RatingInFormComponent', 'IgxCardModule'],
```
Comment (ChronosSF) [inline live-editing/configs/GridConfigGenerator.ts:1465]: This also needs 'IgcFormsModule' to be added to the modules, however, when I try to add it it says:

 Error: Error: IgcFormsModule path is missing for GridWithRatingComponent config!
    at C:\Users\SStoychev\Documents\Git\igniteui-angular-samples\node_modules\igniteui-live-editing\LiveEditingManager.js:65:33

The module is part of igniteui-angular

File: live-editing/configs/GridConfigGenerator.ts [hunk 2]
```diff
@@ -1445,6 +1446,28 @@ export class GridConfigGenerator implements IConfigGenerator {
             })
         }));
 
+        configs.push(new Config({
+            component: 'GridWithRatingComponent',
+            additionalFiles: ['/src/app/directives/prevent-scroll.directive.ts', '/src/app/data/nwindData.ts'],
+            additionalDependencies: ['igniteui-webcomponents'],
+            appModuleConfig: new AppModuleConfig({
+                imports: ['IgxGridModule', 'IgxToastModule', 'GridWithRatingComponent', 'IgxPreventDocumentScrollModule'],
```
Comment (ChronosSF) [inline live-editing/configs/GridConfigGenerator.ts:1454]: See below

**Review by ChronosSF — CHANGES_REQUESTED** [review]

File: live-editing/configs/GridConfigGenerator.ts (hunk 1 above)
Comment (dobromirts) [reply live-editing/configs/GridConfigGenerator.ts]: Resolved

**Review by ChronosSF — APPROVED** [review]

**Pull request merged**